My-Spec-Tutorial.package/RadioButtonGroup.extension/instance/selectedButton.st

```diff
@@ -0,0 +1,4 @@
+*My-Spec-Tutorial
+selectedButton
+
+	^ buttons detect: [ :b | b state ] ifNone: [ nil ]
```

Comment:
Thanks for all of your suggestions. I learned very much from this exercise. Can you tell me what else i can do further.
Also i want to discuss about project Create a smalltalk application for offline text search. So please tell me how can i communicate with you.